### PR TITLE
Coveralls for test coverage

### PR DESCRIPTION
**Pull request by larathordar**
#### Description
Changes proposed in this pull request:

* coveralls added to check that our code is well tested
#### What I have learned working on this feature: 
*synchronise the repos in coveralls to be able to find it

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ gem 'rails', '~> 5.2.0.rc2'
 
 
 group :development, :test do
+  gem 'coveralls', require: false
```
Comment (kmthorsnes) [inline Gemfile:14]: <3

**Review by kmthorsnes — APPROVED** [review]
Looks good to me.

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -5,6 +5,8 @@
 require 'rspec/rails'
 require 'jsonapi/rspec'
 require 'factory_bot'
+require 'coveralls'
```
Comment (davidboll) [inline spec/rails_helper.rb:8]: Shouldn't coveralls be on top?

**Review by davidboll — APPROVED** [review]

File: spec/rails_helper.rb (hunk 2 above)
Comment (tochman) [reply spec/rails_helper.rb:8]: Yes, this needs to go on top of the file.

Comment (holgertidemand) [reply spec/rails_helper.rb:8]: Well this works fine!

**Pull request merged**